src/pypulseq/make_adc.py

```diff
@@ -16,6 +18,9 @@ def make_adc(
     freq_offset: float = 0,
     phase_offset: float = 0,
     system: Union[Opts, None] = None,
+    freq_ppm: float = 0,
+    phase_ppm: float = 0,
+    phase_modulation: np.ndarray = None,
```

Comment:
Should be `Optional[np.ndarray]` or even better `Union[np.ndarray, None]` , which is the (ugly) default syntax we currently use. 

In the future, I would recommend to switch to `np.ndarray | None` syntax , which however requires Python > 3.10. So we have to discuss that first.